db/migrate/20251103151959_add_qr_code_values_to_invoices.rb

```diff
@@ -0,0 +1,43 @@
+class AddQrCodeValuesToInvoices < ActiveRecord::Migration[8.0]
+  def change
+    add_column :invoices, :qr_code_creditor_values, :string, array: true, null: false, default: Array.new(7)
```

Comment:
Das Array gefällt mir nicht so gut, da die Semantik der einzelnen Zeilen nicht in der DB gespeichert ist. Besser fände ich, wenn wir hier mit `serialize` und ev CustomTypes arbeiten analog MailingListe mailchimp_result.

Reply:
Ja, der Array gefällt mir auch nicht. Ein eigens Model fand ich aber auch overkill. Serialize ähnlich wie bei MailchimpResult kannte ich nicht, sieht aber auf den ersten Blick gut aus. Soll ich das so anpassen oder müssen wir das noch besprechen?